template/src/robots.txt.js

```diff
@@ -0,0 +1,5 @@
+module.exports = `
```

Comment:
This can be just the text-file. We could add the `sitemap.xml.js`.

Reply:
Ok